src/main/java/com/google/api/codegen/transformer/OutputTransformer.java

```diff
@@ -267,6 +269,13 @@ public void run() {
 
     if (newVar != null) {
       if (intoScalar) {
+        Preconditions.checkArgument(
+            type.isRepeated() && !type.isMap(),
+            "%s:%s: %s is not a repeated field",
+            context.getMethodModel().getSimpleName(),
+            valueSet.getId(),
+            config.substring(0, end));
+        // "optional" is how protobuf defines singular fields
```

Comment:
nit: I was thinking of this comment after the code on the next line. is there a stylistic reason not to?

Reply:
I think comments usually describe what's about to happen instead of what just happened right? I make a line break to make this more clear.

Reply:
Oh, I meant in the same line:
```
type = type.makeOptional();  // "optional" is how protobuf defines singular fields
```

Sorry I wasn't clear.